tests/test_diffeq/test_odefiltsmooth/test_ivp2filter.py

```diff
@@ -70,36 +58,24 @@ class TestIvp2Ekf1(Ivp2FilterTestCase):
     """
 
     def test_ivp2ekf1_output(self):
-        filtsmooth_object = pnd.ivp2ekf1(self.ivp, self.prior, self.evlvar)
-        self.assertIsInstance(
-            filtsmooth_object.measurement_model, pnfs.DiscreteEKFComponent
+        measmod = pnd.GaussianIVPFilter.string_to_measurement_model(
+            "ekf1", self.ivp, self.prior, self.evlvar
         )
+        self.assertIsInstance(measmod, pnfs.DiscreteEKFComponent)
 
     def test_ekf1_measmod(self):
-        kalman = pnd.ivp2ekf1(self.ivp, self.prior, self.evlvar)
+        measmod = pnd.GaussianIVPFilter.string_to_measurement_model(
+            "eks1", self.ivp, self.prior, self.evlvar
```

Comment:
I call it once each with "ekf1" once and with "eks1" to cover all the possible branches in the dict. Once the tests are refactored, this will be made more clever.